Panic when no Kafka instances were found

After installing rhoas I did the login.
Without creating any new Kafka instance I just tried `rhoas kafka use` and got the following output:

```shell
No Kafka instances were found.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15ee16d]

goroutine 1 [running]:
github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/use.runUse(0xc0001ca0a0, 0xc0001b7c01, 0x8155c5)
	/home/runner/work/app-services-cli/app-services-cli/pkg/cmd/kafka/use/use.go:118 +0x1ad
github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/use.NewUseCommand.func2(0xc000310f00, 0x2667450, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/app-services-cli/app-services-cli/pkg/cmd/kafka/use/use.go:66 +0x85
github.com/spf13/cobra.(*Command).execute(0xc000310f00, 0x2667450, 0x0, 0x0, 0xc000310f00, 0x2667450)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003bd680, 0x1b61f40, 0xd, 0xc0003bd680)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/home/runner/work/app-services-cli/app-services-cli/cmd/rhoas/main.go:70 +0x228
```

--- a/pkg/cmd/kafka/use/use.go
+++ b/pkg/cmd/kafka/use/use.go
@@ -80,6 +80,10 @@ func runUse(opts *Options) error {
 		if err != nil {
 			return err
 		}
+		// no Kafka was selected, exit program
+		if opts.name == "" {
+			return nil
+		}
 	}
 
 	logger, err := opts.Logger()
@@ -115,7 +119,7 @@ func runUse(opts *Options) error {
 
 	// build Kafka config object from the response
 	var kafkaConfig config.KafkaConfig = config.KafkaConfig{
-		ClusterID: *res.Id,
+		ClusterID: res.GetId(),
 	}
 
 	cfg.Services.Kafka = &kafkaConfig

